app.js

```diff
@@ -0,0 +1,20 @@
+const express = require("express")
+const { getTopics } = require("./controllers/nc-news-controllers")
+const app = express()
+
+app.get("/api/topics", getTopics)
+
+
+// error handling
```

Comment:
Comments shouldn't be in the code that lives on the `main` branch, so remember to remove them before doing a PR 😄